mix.exs

```diff
@@ -1,13 +1,13 @@
 defmodule OpenIDConnect.Mixfile do
   use Mix.Project
 
-  @version "0.2.2"
+  @version "1.0.0"
```

Comment:
Nice ;-). Hm, thoughts on maybe reserving the 1.0 for when we pass [OpenID's Relying Party conformance tests](https://openid.net/certification/connect_rp_testing/)?

Don't feel too strongly about, just a thought.

Reply:
We can bump a version to 2.0 later if tests will uncover something to introduce breaking changes, otherwise, it could remain 1.0 or 1.1